evalai/utils/challenges.py

```diff
@@ -2,34 +2,31 @@
 import sys
 
 from bs4 import BeautifulSoup
+from beautifultable import BeautifulTable
 from click import echo, style
 
 from evalai.utils.auth import get_request_header
-from evalai.utils.common import validate_token
+from evalai.utils.common import validate_token, convert_UTC_date_to_local
 from evalai.utils.urls import URLS
 from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
 
 
-def pretty_print_challenge_data(challenge):
+def pretty_print_challenge_data(challenges):
     """
     Function to print the challenge data
     """
-    br = style("----------------------------------------"
-               "--------------------------", bold=True)
-
-    challenge_title = "\n{}".format(style(challenge["title"],
-                                    bold=True, fg="green"))
-    challenge_id = "ID: {}\n\n".format(style(str(challenge["id"]),
-                                       bold=True, fg="blue"))
-
-    title = "{} {}".format(challenge_title, challenge_id)
-
-    cleaned_desc = BeautifulSoup(challenge["short_description"], "lxml").text
-    description = "{}\n".format(cleaned_desc)
-    end_date = "End Date : {}".format(style(challenge["end_date"].split("T")[0], fg="red"))
-    end_date = "\n{}\n\n".format(style(end_date, bold=True))
-    challenge = "{}{}{}{}".format(title, description, end_date, br)
-    echo(challenge)
+    table = BeautifulTable(max_width=200)
+    attributes = ["id", "title", "short_description"]
+    columns_attributes = ["ID", "Title", "Short Description", "Creator", "Start Date", "End Date"]
```

Comment:
Can we also change the color of `column_attributes`? If yes then please try it with green color.

Reply:
@RishabhJain2018 I tried, it breaks the structure of the table.

Reply:
okay!